### PR TITLE
day10

### PR DESCRIPTION
**Pull request by sudheerpvns**

### CODE REVIEW

File: project1/application.py [hunk 1]
```diff
@@ -74,3 +77,29 @@ def admin():
...
+        if result is None:
+            flash("User not found!!!")
+            flash("If you are new user goto Register tab")
+            return render_template("login.html")
+        else:
+            if password == result.password:
+                session["user_id"] = result.user_id   
+                session["user_name"] = result.user_name
+                return render_template("Userhome.html")
+            else:
```
Comment (madhulika9293) [inline project1/application.py:104]: - You have to put login and register buttons in the same register html page in the same form. The idea is to post to different routes when different buttons are clicked.
- Post to '/auth' for login

File: project1/templates/login.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+<h3> Enter your Credentials</h3>
+<form action="{{url_for('login')}}" method ="post">
+<div class = "form">
+    <input = autocomplete="off" autofocus class="form-control" name = "username" placeholder="Username" type = "text"/>
+</div>
+<div class="form">
+    <input class="form-control" name = "password" placeholder="Password" type = "password"/>
+</div>
+<button class ="btn btn-primary">Submit</button>
```
Comment (madhulika9293) [inline project1/templates/login.html:16]: Is it working without input/submit for the button?

**Review by madhulika9293 — COMMENTED** [review]
The functionality seems to be there. Please make sure you adhere to the requirements given in the task description. 
Also, commit changes after each task. I'll review again after the changes.

File: project1/templates/login.html (hunk 2 above)
Comment (sudheerpvns) [reply project1/templates/login.html:16]: yes

**Pull request merged**